src/game/common/system/asciistring.cpp

```diff
@@ -139,6 +159,9 @@ char Utf8String::Get_Char(int index) const
     return Peek()[index];
 }
 
+/**
+ * This is effectively ToString() It is in original code, but never called.
```

Comment:
Its used a lot but is inlined everywhere its called I think, probably in the header originally based on that, but lets not worry about that for now.

Reply:
The message can be edited by whoever knows better. I just go with what i found through what i had.